src/messaging/paircode.ts

```
@@ -0,0 +1,92 @@
+    auth: {
+     creds: state.creds,
+     keys: makeCacheableSignalKeyStore(state.keys, logger),
+    },
+    printQRInTerminal: true,
+    browser: Browsers.ubuntu('Chrome'),
+    logger,
+    version,
+   });
+
+   if (!conn.authState.creds.registered) {
+    let phoneNumber = phone ? phone.replace(/[^0-9]/g, '') : '';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'phoneNumber' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)